src/nccl_ofi_rdma.c

```diff
@@ -2449,7 +2449,7 @@ static int dereg_mr_ep(nccl_net_ofi_rdma_mr_handle_t *mr_handle,
 		} else {
 			ret = nccl_ofi_idpool_free_id(key_pool, key);
 			if (OFI_UNLIKELY(ret != 0)) {
-				NCCL_OFI_WARN("Error freeing MR key %"PRIu64", leaking key", key);
```

Comment:
For the "tree: add spaces around PRIu64" commit, it's that C++ requires whitespace between two string literals that should be concatenated.  Just happens that we mostly do this when one of the string literals is coming from a size identifier.  Your commit message could be less confusing.